pkg/adaptation/adaptation_suite_test.go

```diff
@@ -2008,6 +2025,12 @@ func stripLinuxResourcesUnified(u map[string]string) map[string]string {
 	return u
 }
 
+func stripCDIDevices(a *api.ContainerAdjustment) {
```

Comment:
Just noticed that this seems a bit different to the other functions here since we don't have a return value.

OK. I think the reason it looks strange is that it's not adjacent to the other `strip*` functions for `ContainerAdjustment`. Could we move it?